README.md

```diff
@@ -45,62 +34,78 @@ This is the implementation of the [IPFS repo spec](https://github.com/ipfs/specs
+                                      │                 │
+                                      │        /        │
+                                      │                 │
+                                      ├─────────────────┤
+                                      │    Datastore    │
+                                      └─────────────────┘
+                                               │
+                                   ┌───────────┴───────────┐
+                                   │                       │
+                                   │                       │
+                          ┌─────────────────┐     ┌─────────────────┐
+                          │                 │     │                 │
+                          │     /blocks     │     │   /datastore    │
+                          │                 │     │                 │
+                          ├─────────────────┤     ├─────────────────┤
+                          │    Datastore    │     │ LevelDatastore  │
+                          └─────────────────┘     └─────────────────┘
+
+
+
+┌────────────────────────────────────────┐          ┌────────────────────────────────────────┐
+│       IPFSRepo - Default Node.js       │          │       IPFSRepo - Default Browser       │
+└────────────────────────────────────────┘          └────────────────────────────────────────┘
+
+
+
+            ┌─────────────────┐                                 ┌─────────────────┐
+            │                 │                                 │                 │
+            │        /        │                                 │        /        │
+            │                 │                                 │                 │
+            ├─────────────────┤                                 ├─────────────────┤
+            │   FsDatastore   │                                 │LevelJSDatastore │
+            └─────────────────┘                                 └─────────────────┘
+                     │                                                   │
+         ┌───────────┴───────────┐                           ┌───────────┴───────────┐
+         │                       │                           │                       │
+         │                       │                           │                       │
+┌─────────────────┐     ┌─────────────────┐         ┌─────────────────┐     ┌─────────────────┐
+│                 │     │                 │         │                 │     │                 │
+│     /blocks     │     │   /datastore    │         │     /blocks     │     │   /datastore    │
+│                 │     │                 │         │                 │     │                 │
+├─────────────────┤     ├─────────────────┤         ├─────────────────┤     ├─────────────────┤
+│ FlatfsDatastore │     │LevelDBDatastore │         │LevelJSDatastore │     │LevelJSDatastore │
+└─────────────────┘     └─────────────────┘         └─────────────────┘     └─────────────────┘
```

Comment:
it's a drawing of the datastores and how the are used, what you had before were not actual stores